Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebContext.cpp

```diff
@@ -128,9 +128,13 @@ class URISchemeTest: public LoadTrackingTest {
 
         const char* method = webkit_uri_scheme_request_get_http_method(request);
         g_assert_nonnull(method);
-        if (!g_strcmp0(scheme, "post"))
+        if (!g_strcmp0(scheme, "post")) {
             g_assert_cmpstr(method, ==, "POST");
-        else
+
+            GRefPtr<GBytes> body = webkit_uri_scheme_request_get_http_body(request);
```

Comment:
No need to use GRefPtr here anymore:

```suggestion
            GBytes* body = webkit_uri_scheme_request_get_http_body(request);
```